content/integrations/jira-integration.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Jira Integration
+description: Integrate your workflows with Jira
+weight: 1
+---
+
+**Jira** is a an issue tracking and project management product developed by Atlassian. Many software development teams use it to maintain visibility of their projects. 
+
+It offers a REST API which can be used used in conjunction with your **Codemagic** workflows in order to add comments, upload attachments or transition the status of an issue, story or epic.
+
+The following example shows how to set up integration with Jira using the **codemagic.yaml** configuration with a **native iOS project** but it is the same approach is the same for **Flutter**, **React Native** and **native Android** projects.
+
+## Getting Started
+
+1. Clone the [Jira integratrion starter project](https://github.com/codemagic-ci-cd/jira_integration_demo_project) from the Codemagic GitHub page
+
+2. You will need access to a Jira account and can [sign up](https://www.atlassian.com/software/jira) for free.
+
+3. Create a [Jira API Token](https://id.atlassian.com/manage-profile/security/api-tokens) once you have access to Jira.
+
+## Add the codemagic.yaml to your project
+
+Copy the **codemagic.yaml** from the starter project into the root of your repository. Update the environment variables as indicated and use the documentation links where required for the values related to building your app, code signing and app publishing. 
+
+## Update the Jira environment variables
+
+There are four **environment variables** that need to be updated for the Jira integration: `JIRA_AUTH` `JIRA_BASE_URL` `JIRA_ISSUE` and `JIRA_TRANSITION_ID`
+
+
+#### Updating the JIRA_AUTH environment variable
+
+The `JIRA_AUTH` environment variable is a base64 encoded string which consists of the email address you log into Jira with and the Jira API token you created: 
+
+`email@example.com:<api_token>`
+
+You can encode these credentials in the **macOS Terminal** using:
+
+```
+echo -n 'email@example.com:<api_token>' | openssl base64
+```
+
+Alternatively use an online tool to base64 encode this string. 
+
+This value is used in the Authorization header used in cURL requests to the Jira API.
+
+#### Adding the JIRA_BASE_URL environment variable
+
+This is the subdomain you chose when you set up your Jira account e.g. "YOUR_SUBDOMAIN.atlassian.net". Put the subdomain including 'atalassian.net' in the `JIRA_BASE_URL` environment variable. 
```

Comment:
just wondering if it would make sense to use double or single quotes only for consistency